test/e2e/alerts.e2e.spec.js

```diff
@@ -198,55 +169,71 @@ const main = async () => {
   })
 
   await revokeOtherOAuthClientsForSoftwareId(client, SOFTWARE_ID)
-
-  if (args.push) {
+  if (options.push) {
     const clientInfos = client.stackClient.oauthOptions
     await client.stackClient.updateInformation({
       ...clientInfos,
       notificationPlatform: 'android',
       notificationDeviceToken: 'fake-token'
     })
   }
+  return client
+}
 
-  const allScenarioIds = Object.keys(scenarios)
-  const scenarioIds =
-    args.scenario === 'all'
-      ? allScenarioIds
-      : allScenarioIds.filter(x => x.startsWith(args.scenario))
+describe('alert emails/notifications', () => {
+  let client
+  let pushServer
+  let mailhog
+  let options = {
+    url: process.env.COZY_URL || 'http://cozy.tools:8080',
+    verbose: false
+  }
 
-  const mailhog = args.push ? null : Mailhog({ host: 'localhost' })
-  const pushServer = args.push ? new MockServer() : null
-  if (pushServer) {
+  beforeAll(async () => {
+    pushServer = new MockServer()
     await pushServer.listen()
-  }
+    mailhog = Mailhog({ host: 'localhost' })
+  })
+
+  afterAll(async () => {
+    await pushServer.close()
+  })
 
-  const answers = {}
-  for (const scenarioId of scenarioIds) {
+  beforeEach(async () => {
     await cleanupDatabase(client)
-    const res = await runScenario(client, scenarioId, {
-      showOutput: args.verbose,
-      mailhog,
-      pushServer: pushServer
+  })
+
+  describe('push', () => {
+    beforeAll(async () => {
+      client = await setupClient({ url: options.url })
     })
-    answers[scenarioId] = res
-  }
 
-  for (const [scenarioId, answer] of Object.entries(answers)) {
-    console.log(
-      answer ? '✅' : '❌',
-      scenarioId,
-      `(${scenarios[scenarioId].description})`
-    )
-  }
+    for (const scenario of Object.values(scenarios)) {
+      describe(scenario.description, () => {
+        it('should work', async () => {
```

Comment:
Nit: If there's only one `it` in the scenario, I think we can drop the `describe` and put the scenario description in the `it`. It will remove a nesting level (in test code and in test summary), and also useless "should work".

Reply:
Fixed, thanks !